Doc/library/functions.rst

```diff
@@ -139,7 +140,7 @@ are always available.  They are listed here in alphabetical order.
    See also :func:`format` for more information.
 
 
-.. class:: bool([x])
+.. class:: bool(x=True)
```

Comment:
```suggestion
.. class:: bool(x=False)
```
`bool()` returns `False` and not `True`.

Reply:
That's True.